### DISCUSSION TITLE
commit information as tab or on revision grid

### DISCUSSION THREAD

**Discussion by trojanobelix** [gitextensions/gitextensions#10941] [Q&A] [answered]
This might be a stupid question: How do I get the commit information back into a tab?

I must have accidentally changed a setting, because the commit information is now displayed in the revision grid.

The documentation is not very helpful:

>[Tabs](https://git-extensions-documentation.readthedocs.io/en/release-4.0/settings.html#tabs)

> 
>Commit
>Commit information and message as well as branch and tag information for the selected commit. This tab can be moved to the  revision grid.

**Comment by pmiossec** [comment, marked as answer]
(Not in front of my computer) but it's in the first icons of the toolbar. I think the 4th one... that let you put it at 3 different places.

**Reply by trojanobelix** [reply]
Thank you very much! I've been using GitExtensions for years, but never recognized this icon :-) I tried right clicking on almost every point on the screen ;-)

**Reply by pmiossec** [reply]
But if you have a screen large enough, it could be interesting to have the commit info at the right of the revision grid so that you can read the commit info and the diff of files at the same time without having to switch between tabs.

**Reply by trojanobelix** [reply]
It might be useful to describe the icons of the main bar and their function in a few words. 

I tried and hoped to solve the problem with the help of the documentation.  But without success.

But I know how difficult it is to keep a documentation meaningful and complete. So this is not a criticism, just a hint.

**Reply by RussKie** [reply]
> It might be useful to describe the icons of the main bar and their function in a few words.
> 
> I tried and hoped to solve the problem with the help of the documentation. But without success.
> 
> But I know how difficult it is to keep a documentation meaningful and complete. So this is not a criticism, just a hint.

The docs are located here: https://github.com/gitextensions/GitExtensionsDoc, any help keeping the docs current and relevant is appreciated.
